saltproc/app.py

```diff
@@ -1,98 +1,39 @@
-from saltproc import DepcodeSerpent
-from saltproc import DepcodeOpenMC
-from saltproc import Simulation
-from saltproc import Materialflow
-from saltproc import Process
-from saltproc import Reactor
-from saltproc import Sparger
-from saltproc import Separator
-# from depcode import Depcode
-# from simulation import Simulation
-# from materialflow import Materialflow
-import os
-import copy
+from pathlib import Path
+from copy import deepcopy
+from collections import OrderedDict
+
+import argparse
+import numpy as np
 import json
 import jsonschema
-from collections import OrderedDict
 import gc
 import networkx as nx
 import pydotplus
-import argparse
-import numpy as np
+
+from saltproc import DepcodeSerpent, DepcodeOpenMC, Simulation, Reactor
```

Comment:
is this pep8? I thought it wasn't.

Reply:
[It is](https://peps.python.org/pep-0008/#imports)